config/filesystem.py

```diff
@@ -16,8 +16,6 @@
 STATICFILES = {
     # folder          # template alias
     'storage/static': 'static/',
-    'storage/static/css': 'css/',
-    'storage/compiled/css': 'assets/',
-    'storage/compiled/js': 'js/',
+    'storage/compiled': 'assets/',
```

Comment:
Find this more clear... but opened to discussion.

Reply:
I think this is fine